bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/thing/rm-types.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="broadlink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="rm">
+		<label>Broadlink RM</label>
+		<description>The Broadlink RM Home is a Wi-Fi IR transmitter.</description>
+		<channels>
+			<channel id="command" typeId="command"/>
```

Comment:
Does this have the ability to learn commands? as the channel is missing from here.